.github/workflows/git-artifacts.yml

```diff
@@ -7,14 +7,6 @@ on:
       artifacts:
         description: 'Optionally restrict what artifacts to build (portable, installer, etc.). Separate artifacts with spaces'
         required: false
-      ref:
-        description: 'Optionally override which branch to build'
-        required: false
-        default: main
-      repository:
-        description: 'Optionally override from where to fetch the specified ref'
-        required: false
-        default: git-for-windows/git
```

Comment:
Thanks for removing those! I noticed them earlier this week (copied over from the original pipeline) and was planning to remove them indeed :)